src/controls/first_person.rs

```diff
@@ -1,11 +1,33 @@
 use cgmath;
 use mint;
-use std;
 
 use cgmath::Rotation3;
 use input::{Button, Input, Key};
 use object::Object;
 
+#[derive(Clone)]
+struct FirstPersonKeys {
```

Comment:
This could just be called `Keys` since it's already prefixed by the module name (`first_person::Keys` vs `first_person::FirstPersonKeys`).

Reply:
Uh oh, thanks, I always forget about it